src/Sql/dbo/Stored Procedures/UserSignatureKeyPair_SetForRotation.sql

```diff
@@ -0,0 +1,15 @@
+CREATE PROCEDURE [dbo].[UserSignatureKeyPair_UpdateForRotation]
```

Comment:
The name of this file and the contents of this file don't match. 

`[UserSignatureKeyPair_UpdateForRotation]` code
`src/Sql/dbo/Stored Procedures/UserSignatureKeyPair_SetForRotation.sql` file name.

Reply:
Fixed